### PR TITLE
EVG-15049 add project/repo event log resolvers

### PR DESCRIPTION
**Pull request by ablack12**
[EVG-15049](https://jira.mongodb.org/browse/EVG-15049)

### Description 
Modeled these resolvers off of the host events resolver to allow for pagination.

### Testing 
Not sure how intense the tests need to be since we test ProjectSettings / RepoSettings extensively elsewhere but I added simple tests.

### CODE REVIEW

File: graphql/schema.graphql [hunk 1]
```diff
@@ -1081,21 +1091,44 @@ type GroupedProjects {
 }
 
 type ProjectSettings {
-  githubWebhooksEnabled: Boolean!
+  gitHubWebhooksEnabled: Boolean!
```
Comment (khelif96) [inline graphql/schema.graphql:1094]: This change makes this a breaking change. As long as it isn't being used in production it should be fine.

**Review by khelif96 — APPROVED** [review]
LGTM. Only needs changes if the comment below is true and it is being used in production.

File: graphql/schema.graphql (hunk 1 above)
Comment (ablack12) [reply graphql/schema.graphql:1094]: Not being used in production so should be fine 👍

File: graphql/tests/repoEvents/data.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    }
+  ],
+  "project_vars": [
+    {
+    "_id": "5678",
+    "vars": {"hello": "world", "foo":  "bar"},
+    "private_vars": {"hello":  true, "foo":  false}
+    }
+  ],
+  "event_log": [
```
Comment (sophstad) [inline graphql/tests/repoEvents/data.json:14]: It would be helpful to add this data to the repo_ref file in testdata for local development!

**Review by sophstad — APPROVED** [review]

**Pull request merged**